src/components/switch/Switch.base.tsx

```diff
@@ -0,0 +1,96 @@
+import React from 'react';
+
+import Grid, { Spacing, TSpacingTokensSymmetric } from '../../foundations/layout';
+import Shape from '../../foundations/shape';
+import Text, { TTextSizeToken } from '../text';
+
+import { TSwitchWidths, TSwitchHeights, TSwitchInnerWidths } from './Switch.types';
+import {
+    DEFAULT_SWITCH_HEIGHT,
+    DEFAULT_SWITCH_WIDTH,
+    DEFAULT_SWITCH_INNER_WIDTH,
+    DEFAULT_SWITCH_SIZE,
+} from './Switch.constants';
+import { PSwitch } from './Switch.props';
+
+const SwitchBase: React.FC<PSwitch> = ({
+    labelText = 'Public',
+    size = DEFAULT_SWITCH_SIZE,
+    className,
+    onClick,
+    ...rest
+}: PSwitch) => {
+    let labelSize: TTextSizeToken = 100;
+    let switchHeight: TSwitchHeights = DEFAULT_SWITCH_HEIGHT;
+    let switchWidth: TSwitchWidths = DEFAULT_SWITCH_WIDTH;
+    let switchInnerWidth: TSwitchInnerWidths = DEFAULT_SWITCH_INNER_WIDTH;
+
+    const spacing: TSpacingTokensSymmetric = {
+        vertical: 0,
+        horizontal: 125,
```

Comment:
is this a typo? since the value for the `lg` size is only `25`? Just wondering! :D